config/routes.rb

```diff
@@ -265,6 +265,7 @@
 
         namespace :user do
           root to: 'users#show'
+          get :finance_overview,  to: 'users#finance_overview'
```

Comment:
perhaps "financial_overview" is cleaner here

Reply:
agree